### PR TITLE
Update readme per feedback

### PR DESCRIPTION
**Pull request by timmyc**
Props to @californiakat for providing some edits for our readme text. This PR comprises her feedback. Would like one more set of eyes to ensure I didn't botch anything and will merge and update the svn repo after I get a 👍

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -32,40 +32,42 @@ WooCommerce Admin includes a host of new reports that are optimized for speed an
...
 Visit the [WooCommerce server requirements documentation](https://docs.woocommerce.com/document/server-requirements/) for a detailed list of server requirements.
 
 = Automatic installation =
 
-Automatic installation is the easiest option as WordPress handles the file transfers itself and you don’t need to leave your web browser. To do an automatic install of this plugin, log in to your WordPress dashboard, navigate to the Plugins menu and click Add New.
-
-In the search field type “WooCommerce Admin” and click Search Plugins. Once you’ve found this plugin you can view details about it such as the point release, rating and description. Most importantly of course, you can install it by simply clicking “Install Now”.
+Automatic installation is the easiest option, as WordPress handles the file transfers and you need not leave your web browser. To perform an automatic install: 
+1. Log in to your WordPress dashboard.
+2. Go  to:Plugins > Add New.
```
Comment (Aljullu) [inline readme.txt]: Missing space here?
```suggestion
2. Go to: Plugins > Add New.
```

File: readme.txt [hunk 2]
```diff
@@ -32,40 +32,42 @@ WooCommerce Admin includes a host of new reports that are optimized for speed an
...
+* MySQL version 5.0 or greater. MySQL 5.6 or greater is recommended
 
 Visit the [WooCommerce server requirements documentation](https://docs.woocommerce.com/document/server-requirements/) for a detailed list of server requirements.
 
 = Automatic installation =
 
-Automatic installation is the easiest option as WordPress handles the file transfers itself and you don’t need to leave your web browser. To do an automatic install of this plugin, log in to your WordPress dashboard, navigate to the Plugins menu and click Add New.
-
-In the search field type “WooCommerce Admin” and click Search Plugins. Once you’ve found this plugin you can view details about it such as the point release, rating and description. Most importantly of course, you can install it by simply clicking “Install Now”.
+Automatic installation is the easiest option, as WordPress handles the file transfers and you need not leave your web browser. To perform an automatic install:
```
Comment (Aljullu) [inline readme.txt]: Is that grammatically correct "you need not leave your web browser"?

File: readme.txt [hunk 3]
```diff
@@ -32,40 +32,42 @@ WooCommerce Admin includes a host of new reports that are optimized for speed an
...
 
 = Automatic installation =
 
-Automatic installation is the easiest option as WordPress handles the file transfers itself and you don’t need to leave your web browser. To do an automatic install of this plugin, log in to your WordPress dashboard, navigate to the Plugins menu and click Add New.
-
-In the search field type “WooCommerce Admin” and click Search Plugins. Once you’ve found this plugin you can view details about it such as the point release, rating and description. Most importantly of course, you can install it by simply clicking “Install Now”.
+Automatic installation is the easiest option, as WordPress handles the file transfers and you need not leave your web browser. To perform an automatic install: 
+1. Log in to your WordPress dashboard.
+2. Go  to:Plugins > Add New.
+3. Enter “WooCommerce Admin” in the Search field, and  view details about its point release, the rating and description.
```
Comment (Aljullu) [inline readme.txt]: There is a double-space here:
```suggestion
3. Enter “WooCommerce Admin” in the Search field, and view details about its point release, the rating and description. 
```

File: readme.txt [hunk 4]
```diff
@@ -32,40 +32,42 @@ WooCommerce Admin includes a host of new reports that are optimized for speed an
...
 = Automatic installation =
 
-Automatic installation is the easiest option as WordPress handles the file transfers itself and you don’t need to leave your web browser. To do an automatic install of this plugin, log in to your WordPress dashboard, navigate to the Plugins menu and click Add New.
-
-In the search field type “WooCommerce Admin” and click Search Plugins. Once you’ve found this plugin you can view details about it such as the point release, rating and description. Most importantly of course, you can install it by simply clicking “Install Now”.
+Automatic installation is the easiest option, as WordPress handles the file transfers and you need not leave your web browser. To perform an automatic install: 
+1. Log in to your WordPress dashboard.
+2. Go  to:Plugins > Add New.
+3. Enter “WooCommerce Admin” in the Search field, and  view details about its point release, the rating and description. 
+4. Select  “Install Now” when you’re ready
```
Comment (Aljullu) [inline readme.txt]: And here too. :upside_down_face: 
```suggestion
4. Select “Install Now” when you’re ready
```

File: readme.txt [hunk 5]
```diff
@@ -32,40 +32,42 @@ WooCommerce Admin includes a host of new reports that are optimized for speed an
...
 
 Bugs should be reported in the [WooCommerce Admin GitHub repository](https://github.com/woocommerce/woocommerce-admin/).
 
 = This is awesome! Can I contribute? =
 
-Yes you can! Join in on our [GitHub repository](https://github.com/woocommerce/woocommerce-admin/) :)
+Yes, you can! Join our [GitHub repository](https://github.com/woocommerce/woocommerce-admin/)
 
-Release and roadmap notes available on the [WooCommerce Developers Blog](https://woocommerce.wordpress.com/2019/01/15/woocommerce-blocks-1-3-0-release-notes/)
+Release and roadmap notes are available on the[WooCommerce Developers Blog](https://woocommerce.wordpress.com/2019/01/15/woocommerce-blocks-1-3-0-release-notes/)
```
Comment (Aljullu) [inline readme.txt]: We should keep that space, right?
```suggestion
Release and roadmap notes are available on the [WooCommerce Developers Blog](https://woocommerce.wordpress.com/2019/01/15/woocommerce-blocks-1-3-0-release-notes/)
```

**Review by Aljullu — APPROVED** [review]
Text looks good, just some comments/questions.

**Comment by timmyc** [comment]
Thanks for the proof reading @Aljullu - I really should have caught those but figured the Google Doc would have been fine 🤦‍♂️

**Comment by californiakat** [comment]
When you go through and accept changes in a Google doc with suggestions, it needs a once-over to make sure stray spaces aren't still around.

**Pull request merged**